tests/regression_tests/ncrystal/test.py

```diff
@@ -69,12 +69,31 @@ def _get_results(self):
             df = tal.get_pandas_dataframe()
         return df.to_string()
 
+def test_cfg_from_xml(model):
+     """Make sure the cfg string is read by from_xml method"""
+
+    #export the original material generated with cfg string
+    model.materials.export_to_xml('materials.xml.orig')
+    expected = open('materials.xml.orig', 'r').readlines()
+    #read back the original material
+    mats_from_xml = openmc.Materials.from_xml('materials.xml.orig')
+    #export again
+    mats_from_xml.export_to_xml('materials.xml.after')
+    actual = open('materials.xml.after', 'r').readlines()
+    compare = filecmp.cmp('materials.xml.orig','materials.xml.after')
+    if not compare:
+        diff = unified_diff(expected, actual, 'materials.xml.orig',
+                           'materials.xml.after')
+        print('Input differences:')
+        print(''.join(diff))
+    assert compare, 'Materials not read correctly from XML'
 
 def test_ncrystal():
     n_particles = 100000
     T = 293.6  # K
     E0 = 0.012  # eV
     cfg = 'Al_sg225.ncmat'
     test = pencil_beam_model(cfg, E0, n_particles)
+    test_cfg_from_xm(test)
```

Comment:
You have a typo here for one (missing "l" at the end of "xml"). Also, it would be good for the new test to be separate from `test_ncrystal`. You could do this by simply removing the call here, removing the `model` argument on `test_cfg_from_xml`, and then simply call `pencil_beam_model` from within `test_cfg_from_xml`. That way, we will get two different test results reported rather than one.

Reply:
Got it! Thanks for the help and sorry for the silly style mistakes